woocommerce/woocommerce
# CYS -- Intro design updates

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR updates the CYS intro page to reflex the latest design changes.

1. Replaced the banner with the latest one
2. Added banner and text copy for offline mode
3. Various style fixes
4. Added `Active theme` badge
5. Added color color palettes

This PR doesn't handle theme click event, but `ThemeCard` component accepts `link` prop. Once we have changes from https://github.com/woocommerce/woocommerce/issues/40482, we should refactor the code. 

This PR doesn't handle Jetpack connection design yet. You can still test it by flipping https://github.com/woocommerce/woocommerce/pull/40503/files#diff-0c89278189ca36a2aa455c6193226c3abd4b76a4bb5a78d692ea8c11b2bad495R40 variable



### How to test the changes in this Pull Request:

This feature is part of the larger customize your store development and behind a feature flag, hence QA review can be deferred until the call for testing before release.

Make sure to install & active WooCommerce Beta Tester and turn on `customize-store` feature flag from `Tools -> WCA Test Helper -> Features`

1. Create a new JN site with this branch.
2. Complete the core profiler
3. Go to `WooCommerce Home` and click on `Customize your store` task
4. Turn off your WIFI
5. You should see the offline banner and copy.

![Screen Shot 2023-09-28 at 2 05 23 PM](https://github.com/woocommerce/woocommerce/assets/4723145/f2095539-8aaa-4c46-a487-6fdc6e6dad18)
6. Compare the page with the Figma and confirm the design matches.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

CYS - Intro page design updates

#### Comment 

</details>
